src/watchmaker/static/config.yaml

```
@@ -36,7 +36,7 @@ linux:
             - redhat
             - centos
           el_version: 7
-          url: https://watchmaker.cloudarmor.io/yum.defs/saltstack/salt/3003.3/salt-reposync-el7-python3.repo
+          url: https://watchmaker.cloudarmor.io/watchmaker/yum.defs/saltstack/salt/3004/salt-reposync-el7-python3.repo
```

while this works, prefer leaving out the leading "watchmaker" prefix

```suggestion
          url: https://watchmaker.cloudarmor.io/yum.defs/saltstack/salt/3004/salt-reposync-el7-python3.repo
```